backend/spring-routie/src/main/java/routie/routiespace/controller/dto/request/PlaceCreateRequest.java

```diff
@@ -0,0 +1,19 @@
+package routie.routiespace.controller.dto.request;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import jakarta.validation.constraints.NotNull;
+import java.time.DayOfWeek;
+import java.time.LocalTime;
+import java.util.List;
+
+public record PlaceCreateRequest(
+        String name,
+        String address,
+        int stayDurationMinutes,
+        @NotNull @JsonFormat(pattern = "HH:mm") LocalTime openAt,
+        @NotNull @JsonFormat(pattern = "HH:mm") LocalTime closeAt,
+        @JsonFormat(pattern = "HH:mm") LocalTime breakStartAt,
+        @JsonFormat(pattern = "HH:mm") LocalTime breakEndAt,
```

Comment:
검증이 추가될 때마다 변수 이름이 점점 오른쪽으로 밀려서 확인이 느려지는 것 같다는 생각이 들어요.
아래와 같이 검증 어노테이션에서 개행 후, 변수를 선언하는 건 어떨까요?
```suggestion
        @NotNull @JsonFormat(pattern = "HH:mm") 
        LocalTime openAt,
        @NotNull @JsonFormat(pattern = "HH:mm") 
        LocalTime closeAt,
        @JsonFormat(pattern = "HH:mm") 
        LocalTime breakStartAt,
        @JsonFormat(pattern = "HH:mm") 
        LocalTime breakEndAt,
```